### PR TITLE
Add content to episode 1 of the tutorial

### LINKED ISSUE #28: Episode 1 - tutorial sprint
To do:

- [x]  Copy outline from [document](https://docs.google.com/document/d/1eT7laRvxmI0OhFIj6A9PRhXQIAOHg_MVzz4X_oV-28Q/edit#) to the tutorial
- [x] Change/extend the outline
- [x] Add content (still need to add section on "the community")
- [ ] Check that all questions are anwered / objectives addressed.
- [ ] ...

### PR DESCRIPTION
**Pull request by Peter9192**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #28

### CODE REVIEW

**Comment by Peter9192** [comment]
@SarahAlidoost I did this work on and off together with @JaroCamphuijsen. So he's seen most of it, but maybe another look would be good. Do you have time, or know who can do it?

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -1,15 +1,116 @@
...
+> >
+> > &#10003; **Free.** ESMValTool is licenced under Apache 2.0, which means everyone can use, modify, or share it free of charge. However, we *do* encourage all users to contribute to the community once they get more comfortable with the tool.
+> >
+> > &#10003; **A command line tool.** ESMValTool was originally designed for the command line. But, we are working on a user-friendly python interface as well.
+> >
+> > &#10003; **A way to make climate science more [FAIR](https://fair-software.eu/about).** ESMValTool collects provenance information about the data and code that are used to obtain a result. It comes with a readable recipe format that makes climate analysis consistent, reproducible, and easy to share.
+> >
+> > &#10005;  **Perfect.** Although we are continuously working to improve the tool, you may encounter some bugs or missing features. In this lesson, you will learn how to troubleshoot, find help, and maybe even contribute to the solution yourself.
```
Comment (hb326) [inline _episodes/01-introduction.md:57]: I am not sure that everybody knows that a "lesson" consists of several episodes. Maybe we could rephrase the second sentence to "In the following episodes, you will learn..." to make sure that nobody expects to understand troubleshooting etc. after the first episode.

**Comment by hb326** [comment]
The same figure that is used in this episode is also used in the episode about the preprocessor (episode 5). Maybe we can store the figure somewhere central so that both episodes can access it, and we don't have to store it twice?

File: _episodes/01-introduction.md [hunk 2]
```diff
@@ -1,15 +1,116 @@
...
+> >
+> > &#10005;  **Suitable for (Jupyter) notebooks.** ESMValTool was designed as a command line tool. But, we are working on a user-friendly Python interface as well.
+> {: .solution}
+{: .challenge}
+
+To learn more about ESMValTool, you can look at the [documentation](https://docs.esmvaltool.org/en/latest/introduction.html), the [official website](https://www.esmvaltool.org/about.html), or the [overview paper](https://gmd.copernicus.org/articles/13/1179/2020/) in *Geoscientific Model Development*.
+
+## How does ESMValTool work
+
+The figure below shows the different components of ESMValTool. Most of the work is done in the 'core', which performs a number of preprocessor steps. Outside of the core we have some configuration files that specify things like *where to find the CMIP data*. The most important file however, is the *recipe* that specifies which preprocessor functions need to be applied to what data. The recipe also points to a diagnostic script that is executed after the preprocessor and performs a more specific analysis on the preprocessed data.
```
Comment (hb326) [inline _episodes/01-introduction.md:67]: I am not sure that the statement "most of the work is done in the 'core'" is correct. It totally depends on how complex the diagnostics are. And sometimes there is no preprocessing done at all, so there is actually not that much work done in the core. Maybe we can rephrase to "some of the most important work is done in the 'core'" or something like that?

File: _episodes/01-introduction.md [hunk 3]
```diff
@@ -1,15 +1,116 @@
...
+> > - Recipes are easy to read and share.
+> > - A collection of recipes and diagnostic scripts is shipped with ESMValTool, ready for re-use. Everyone can add to this collection.
+> > - The recipe format takes some getting used to and may be a bit less flexible then working on the datasets directly.
+> > - Features or dataset support that are missing from ESMValCore can be a limiting factor.
+> {: .solution}
+{: .discussion}
+
+## Community
+
+ESMValTool is built and maintained by an active community of scientists and engineers. Many of the interactions take place on GitHub. Here, we briefly introduce you to some of the most important pages.
```
Comment (hb326) [inline _episodes/01-introduction.md:90]: Do you want to specify the engineer type here as "software engineers"?

File: _episodes/01-introduction.md [hunk 4]
```diff
@@ -1,15 +1,116 @@
...
+
+> ## Challenge: meet ESMValGroup
+>
+> Browse to [github.com/ESMValGroup](https://github.com/ESMValGroup). This is our 'organization' GitHub page. Have a look around. How many collaborators are there? Do you know any of them? You should see 2 main repositories: ESMValTool and ESMValCore. Clicking on them will take you to there. How many people have contributed to each of them? Can you also find out how many people have contributed to this tutorial?
```
Comment (hb326) [inline _episodes/01-introduction.md:94]: I think I would rephrase "Clicking on them will take you there".

File: _episodes/01-introduction.md [hunk 5]
```diff
@@ -1,15 +1,116 @@
...
+>
+> > ## Solution
+> >
+> > At the time of making this lesson, there were 138 members of ESMValGroup. 55 of them contributed to ESMValTool, 48 to ESMValCore. 52 (!) people contributed to this tutorial.
```
Comment (hb326) [inline _episodes/01-introduction.md:98]: I like that there are actual numbers given here, but I can also see that this might be a problem since it might not automatically update and therefore the numbers could be "old" pretty fast.

Comment (hb326) [reply _episodes/01-introduction.md:98]: Maybe we have to add a date here so that it is clear when we were actually checking the numbers.

Comment (hb326) [reply _episodes/01-introduction.md:98]: But maybe the statement "At the time of making this lesson..." might be enough.

File: _episodes/01-introduction.md [hunk 6]
```diff
@@ -1,15 +1,116 @@
...
+> {: .solution}
+{: .challenge}
+
+> ## Challenge: issues and pull requests
+>
+> Go back to the repository pages of [ESMValTool](https://github.com/ESMValGroup/ESMValTool) or [ESMValCore](https://github.com/ESMValGroup/ESMValCore). You should see tabs for 'issues' and 'pull requests'. You can use the labels to navigate them a bit more. How many open issues are about enhancements of ESMValTool? And how many bugs have been fixed in ESMValCore? There is also an 'insights' tab, where you can see a summary of recent activity. How many issues have been opened and closed in the past month?
+>
+> > ## Solution
+> >
+> > At the time of making this tutorial, there were 50 ESMValTool issues (the majority) about enhancement and 31 bugs had been fixed in the Core. 13 ESMValTool issues had been closed in the past month, versus 8 opened: overall good progress.
```
Comment (hb326) [inline _episodes/01-introduction.md:108]: See comment above about the exact numbers.

Comment (hb326) [reply _episodes/01-introduction.md:108]: Maybe we should adjust the half sentence "At the time of making this tutorial, ..." to the identical to the one before "At the time of making this lesson..." or vice versa. For consistency in the nomenclature of the "lesson" or "episode"or "tutorial".

**Comment by Peter9192** [comment]
@hb326 Thanks for your suggestions, I implemented them. Indeed, the second one between this episode and episode 5 that get's merged should refer to the existing figure. 

PS sorry for putting the wrong name in my commit message

**Comment by hb326** [comment]
> 
> 
> @hb326 Thanks for your suggestions, I implemented them. Indeed, the second one between this episode and episode 5 that get's merged should refer to the existing figure.
> 
> PS sorry for putting the wrong name in my commit message

No worries!  :smile:

File: _episodes/01-introduction.md [hunk 7]
```diff
@@ -1,15 +1,116 @@
...
+> {: .solution}
+{: .discussion}
+
+## Community
+
+ESMValTool is built and maintained by an active community of scientists and software engineers. Many of the interactions take place on GitHub. Here, we briefly introduce you to some of the most important pages.
+
+> ## Challenge: meet ESMValGroup
+>
+> Browse to [github.com/ESMValGroup](https://github.com/ESMValGroup). This is our 'organization' GitHub page. Have a look around. How many collaborators are there? Do you know any of them? You should see 2 main repositories: ESMValTool and ESMValCore. Visit each of the repositofires. How many people have contributed to each of them? Can you also find out how many people have contributed to this tutorial?
```
Comment (sverhoeven) [inline _episodes/01-introduction.md:94]: Typo
```suggestion
> Browse to [github.com/ESMValGroup](https://github.com/ESMValGroup). This is our 'organization' GitHub page. Have a look around. How many collaborators are there? Do you know any of them? You should see 2 main repositories: ESMValTool and ESMValCore. Visit each of the repositories. How many people have contributed to each of them? Can you also find out how many people have contributed to this tutorial?
```

**Comment by Peter9192** [comment]
Hey @SarahAlidoost . Thanks for the feedback. I removed the 'shoulds' and the 'challenges'. However, I disagree about avoiding a 'challenge' in the beginning. I think this is a nice, interactive way to synchronize expectation and get a first feel, and I'd like to keep it like this.
